packages/shared/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@redhat-cloud-services/shared",
```

Comment:
This name is extremely generic. We should make it explicit that is for the JS client. It needs to be more specific. Something like:
```js
"@redhat-cloud-services/javascript-clients-shared
```